gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/utils/ConstantKeys.java

```diff
@@ -41,6 +41,7 @@
     String ERROR_DESCRIPTION_PARAM_KEY = "error_description";
     String SUCCESS_PARAM_KEY = "success";
     String WARNING_PARAM_KEY = "warning";
+    String PASSWORD_CONFIG_PARAM_KEY = "passwordConfig";
```

Comment:
Replace passwordConfig for passwordSettings (to keep consistency with the backend).

Reply:
OK